ledger/participant-state/protobuf/com/daml/ledger/participant/state/ledger_configuration.proto

```diff
@@ -31,6 +31,9 @@ message LedgerConfiguration {
   // The ledger time model, specifying the bounds for
   // ledger effective time and maximum record time of transactions.
   LedgerTimeModel time_model = 3;
+
+  // The maximum time window during which commands can be deduplicated.
```

Comment:
Might be a good idea to expand how this works here. At the very least, document the semantics of this field in https://github.com/digital-asset/daml/blob/master/ledger/participant-state/protobuf/ledger_configuration.rst. Also increment the version number and make sure the decoder correctly picks a default if the field is unset.